akka-docs/src/main/paradox/persistence-query.md

```diff
@@ -29,10 +29,8 @@ side of an application, however it can help to migrate data from the write side
 simple scenarios Persistence Query may be powerful enough to fulfill the query needs of your app, however we highly
 recommend (in the spirit of CQRS) of splitting up the write/read sides into separate datastores as the need arises.
 
-
-The [CQRS with Akka 2.6 video](https://akka.io/blog/news/2020/02/05/akka-cqrs-video) is a good starting point for
-learning how to use `eventsByTag` to implement CQRS with Akka. Also, watch the introduction to 
-[Event Sourcing with Akka 2.6 video](https://akka.io/blog/news/2020/01/07/akka-event-sourcing-video).
+The @extref[Microservices with Akka tutorial](platform-guide:microservices-tutorial/index.html) explains how to
```

Comment:
IIRC google likes the links slightly better without the `index.html`
```suggestion
The @extref[Microservices with Akka tutorial](platform-guide:microservices-tutorial/) explains how to
```

Reply:
ok, changed